cddagl/constants.py

```diff
@@ -72,6 +72,16 @@
 }
 
 STABLE_ASSETS = {
+    '0.F-3': {
+        'name': '0.F Frank-3',
+        'number': '11002',
+        'released_on': '2021-11-27T10:16:25Z',
+        'github_release': 'https://github.com/CleverRaven/Cataclysm-DDA/releases/tag/0.F-3',
+        'Tiles': {
+            'x64': 'https://github.com/CleverRaven/Cataclysm-DDA/releases/download/0.F-3/cataclysmdda-0.F-Windows_x64-Tiles-0.F-3.zip',
+            'x86': 'https://github.com/CleverRaven/Cataclysm-DDA/releases/download/0.F-3/cataclysmdda-0.F-Windows-Tiles-0.F-3.zip'
+        }
+    },
     '0.F-2': {
         'name': '0.F Frank-2',
         'number': '2021-08-31-2315',
```

Comment:
It seems that the "number" tag was, like you said, the build number but later was changed to the "Date of release" in a YYYY-MM-DD-HHMM format (CEST -2 time). Where did you find that version.txt file?

Reply:
In the root of the Windows-Tiles zip files (so the installed game files)

Reply:
Thanks!